src/Client/AzureManaged/Client.AzureManaged.csproj

```
@@ -12,7 +12,7 @@
 
   <ItemGroup>
     <PackageReference Include="Azure.Identity" Version="1.13.1" />
-    <PackageReference Include="Microsoft.Extensions.Options.DataAnnotations" Version="8.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Options.DataAnnotations" Version="7.0.0" />
```

This needs to be `6.x`
```suggestion
    <PackageReference Include="Microsoft.Extensions.Options.DataAnnotations" Version="6.0.1" />
```